projects/canopy/src/styles/mixins.scss

```diff
@@ -69,14 +69,12 @@
   border-bottom: 0;
   padding: 0;
 
-  &:hover {
-    border-bottom: 0;
-    box-shadow: none;
-  }
-
+  &:hover,
   &:active,
   &:focus,
   &:focus:hover {
```

Comment:
Now if I'm focused on a link and hover over it the focus disappear, it doesn't look right?

Reply:
Tested on the breadcrumb: https://deploy-preview-466--legal-and-general-canopy.netlify.app/?path=/story/components-breadcrumb--two-items

Reply:
Great spot, have updated the styles 👍